tests/datasets/test_biomassters.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+
+import os
+import shutil
+from itertools import product
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pytest
+from _pytest.fixtures import SubRequest
+from pytest import MonkeyPatch
+
+from torchgeo.datasets import BioMassters
+
+
+class TestBioMassters:
+    @pytest.fixture(
+        params=product(["train", "test"], [["S1"], ["S2"], ["S1", "S2"]], [True, False])
+    )
+    def dataset(
+        self, monkeypatch: MonkeyPatch, tmp_path: Path, request: SubRequest
+    ) -> BioMassters:
+        split, sensors, as_time_series = request.param
+
+        url = os.path.join("tests", "data", "biomassters", "{}")
+        monkeypatch.setattr(BioMassters, "url", url)
+
+        # copy data for now
+        shutil.rmtree(tmp_path)
+        shutil.copytree(os.path.join("tests", "data", "biomassters"), str(tmp_path))
```

Comment:
What if we drop `tmp_path` from the function entirely and just give it the `tests/data/biomassters` directory as the root dir? Then we don't need to bother copying data.